### ISSUE TITLE
page numbers

### ISSUE THREAD

**Issue by rettinghaus** [TEI-Correspondence-SIG/CMIF#7]
Although most scholarly editions letters are numbered consecutively, but sometimes one wants to give a page number. If simply put it in the key attribute, there'd no way to differentiate between letter and page numbers.

**Comment by StefanDumont** [comment]
Yes, that's right. Any ideas how we can distinguish one from another?

* Using another attribute, like `@n`
* Using one or more specific character(s) before the page number? Like `p#`
